src/algorithms/bubble_sort.rs

```diff
@@ -0,0 +1,18 @@
+// function to sort an array of integers using bubble sort
+pub fn sort(arr: &mut [i32]) {
+    let mut i = 0;
+    let mut j = 0;
+    let mut temp;
+    while i < arr.len() {
+        while j < arr.len() - 1 {
+            if arr[j] > arr[j + 1] {
+                temp = arr[j];
+                arr[j] = arr[j + 1];
+                arr[j + 1] = temp;
```

Comment:
@Segmentree check this out this link https://doc.rust-lang.org/stable/std/mem/fn.swap.html to be able to swap in one line

Reply:
@rbruno95 seems to have a problem when you attempt to swap to array elements.

https://stackoverflow.com/questions/28294735/how-to-swap-the-elements-of-an-array-slice-or-vec

Reply:
@rbruno95 forget about it you can call the swap function in the array class and pass two indexes.